common/viewer.cpp

```diff
@@ -1303,6 +1303,16 @@ namespace rs2
         return res;
     }
 
+    void force_minimum_size_for_display(rs2::stream_model& model)
+    {
+        // patch for safety sensor
+        if (model.profile.stream_type() == RS2_STREAM_SAFETY)
+        {
+            model.size.x = 7.f;
```

Comment:
Please elaborate on the magic values here.
Why 7,
Why 10
Comment?

Reply:
done